bootstrap/092_service_account_setup.sql

```diff
@@ -0,0 +1,29 @@
+
+CREATE OR REPLACE PROCEDURE admin.finalize_setup_from_service_account(token varchar, url varchar, web_url varchar)
+RETURNS varchar
+LANGUAGE sql
+as
+begin
+    execute immediate 'create or replace function internal.get_ef_url() returns string as \'\\\'' || url || '\\\'\';';
+    execute immediate 'create or replace function internal.get_tenant_url() returns string as \'\\\'' || web_url || '\\\'\';';
+
+    call internal.set_config('tenant_url', web_url);
+    call internal.set_config('url', url);
+    call admin.update_reference('OPSCENTER_API_INTEGRATION', 'ADD', SYSTEM$REFERENCE('API Integration', 'OPSCENTER_API_INTEGRATION', 'persistent', 'usage'));
+    call admin.connect_sundeck(token);
+    call admin.finalize_setup();
```

Comment:
Can we call this via the task so it happens in the backend to avoid this procedure taking a long time?

Reply:
as discussed Ive removed finalize and connect sundeck. The assumption being that these will be called by the linking service